### PR TITLE
CI: Add first draft for schema molecule coverage

### PR DESCRIPTION
**Pull request by gmuloc**
## Change Summary

Add a pre-commit hook to compute schema coverage of molecule tests

## Component(s) name

`ci`

## Proposed changes

* remaining work:
    * eos_designs
    * document code

## How to test

run molecule

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: .pre_commit_scripts/current_coverage.yml
Comment (ClausHolbechArista) [inline .pre_commit_scripts/current_coverage.yml:1]: Adding this as a file is not very friendly for contributors not running pre-commit locally. They can do most other things manually in some way or another, but this one requires pre-commit to run. So every time you change a molecule scenario you have to rerun this.
Maybe we can run this in CI on `devel` first, before checking out PR branch, and then run it again, submitting a comment on the PR with the before and after.

Comment (gmuloc) [reply .pre_commit_scripts/current_coverage.yml:1]: they can run it using the python script as well - if we have to we can make a makefile entry .. :) 
waiting to push to get feedback is not really better

Comment (gmuloc) [reply .pre_commit_scripts/current_coverage.yml:1]: this is the exact same as for the schemas hook actually

**Comment by ClausHolbechArista** [comment]
This will be covered (ha) by pytest "coverage" over time, so I think we can skip this.

**Pull request closed without merging**